app/modules/search/components/global_search_bar.svelte

```diff
@@ -88,7 +91,9 @@
         limit: searchBatchLength,
         offset: searchOffset,
       })
-      hasMore = res.continue != null
+      hasMore = res.continue < resultsMaxLength
```

Comment:
Fun JS fact I prefer not to have to think about
```js
null < 100 // true
undefined < 100 // false
```
```suggestion
      hasMore = res.continue != null && res.continue < resultsMaxLength
```

Reply:
thx, this cannot stick in my head. I made a note to try to not forget. Fixed in 1c22ef762